Dockerfile

```
@@ -54,7 +70,22 @@ RUN npm run fix-web-dev
 
 # Stage 2: Serve the app using Nginx.
 FROM nginx:alpine
+
+# Remove the default Nginx configuration
+RUN rm /etc/nginx/conf.d/default.conf
+
+# Copy SSL certificate from the build stage
+COPY --from=build /usr/src/app/server.key /etc/nginx/conf.d/server.key
+COPY --from=build /usr/src/app/server.crt /etc/nginx/conf.d/server.crt
```

Related to previous commend

```suggestion
COPY --from=cert-gen /certs /etc/nginx/conf.d
```